crates/matrix-sdk-sqlite/src/crypto_store.rs

```diff
@@ -785,7 +785,7 @@ trait SqliteObjectCryptoStoreExt: SqliteAsyncConnExt {
     ) -> Result<Option<Vec<u8>>> {
         Ok(self
             .query_row(
-                "SELECT bundle_data FROM received_room_key_bundle WHERE room_id = ? AND sender_user = ?",
+                "SELECT bundle_data FROM received_room_key_bundle WHERE room_id = ? AND sender_user_id = ?",
```

Comment:
This is a bug from a previous shared-history PR: https://github.com/matrix-org/matrix-rust-sdk/pull/4932/files#diff-30a86db111e1519854209ad10ea46e280484732d1a5e3d409af6a7aa02303b1fR788

We're now triggering this codepath in various unrelated tests and it fails. I'll add a dedicated test for this codepath in a separate PR.